src/codegen/compile.ml

```diff
@@ -5874,7 +5874,7 @@ module BlobStream : Stream = struct
   let name_for fn_name ts = "@Bl_" ^ fn_name ^ "<" ^ Typ_hash.typ_seq_hash ts ^ ">"
 
   let absolute_offset env get_token =
```

Comment:
```suggestion
  let absolute_offset env get_token =
```
Maybe rename this to avoid confusion with rebound `absolute_offset` below?

Reply:
(or rename the other one)

Reply:
It is part of the module `sig`nature. I cannot rename just one.